### PR TITLE
Fix SSF-122

### LINKED ISSUE #2123: Fix SSF-122

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**
Fixes #2123

### CODE REVIEW

File: eslint-bridge/tests/server.test.ts [hunk 1]
```diff
@@ -338,7 +338,7 @@ describe('should send error when failing', () => {
   let close;
 
   beforeEach(async () => {
-    server = await startServer(0, failAnalysis, failAnalysis);
+    server = await startServer(0, '127.0.0.1', failAnalysis, failAnalysis);
```
Comment (andrea-guarino-sonarsource) [inline eslint-bridge/tests/server.test.ts:341]: can we add a test where the loopback address is not '127.0.0.1'?

**Review by andrea-guarino-sonarsource — COMMENTED** [review]
I would like to have an (integration?) test asserting that the node server is only listening on 127.0.0.1 and not on any other ip addresses. 
However I'm not sure this is very easy to test: if you think it's too complicated, I'm ok with trusting that `app.listen` will behave as expected.

**Comment by yassin-kammoun-sonarsource** [comment]
> * I would like to have an (integration?) test asserting that the node server is only listening on 127.0.0.1 and not on any other ip addresses.

As discussed during standup, it seems quite trick to write such a test. Let's trust Express.js on that one.

> * why is the coverage at 87.5 %? I couldn't understand from SQ UI

It was due to parameters with default values. If default values are not used, the test framework considers that branches are not covered.

File: eslint-bridge/tests/server.test.ts (hunk 1 above)
Comment (yassin-kammoun-sonarsource) [reply eslint-bridge/tests/server.test.ts]: I don't think we can. At least, for the particular case of macOS, it does not work unless you manually create some kind of alias (see [here](https://superuser.com/questions/458875/how-do-you-get-loopback-addresses-other-than-127-0-0-1-to-work-on-os-x)).

**Review by andrea-guarino-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**